tests/integration/oci-image-charm/metadata.yaml

```diff
@@ -0,0 +1,10 @@
+name: oci-image-charm
+series: ["focal"]
+summary: "test"
+description: "test"
+maintainers: ["test"]
+resources:
+  oci-image:
+    type: oci-image
+    description: 'Backing OCI image'
+    auto-fetch: true
```

Comment:
This doesn't do anything and should be dropped. (It is used by a helper library in reactive charms, but that's not applicable here.)